src/components/HeaderWithLanguage/HeaderActions.tsx

```diff
@@ -83,6 +81,20 @@ const StyledGroup = styled.div`
   display: flex;
   align-items: center;
 `;
+const StyledSafeLinkButton = styled(SafeLinkButton)`
```

Comment:
No er denne definert _under_ der den brukes. Typescript klager kanskje ikkje på det?

Reply:
Gjorde det slik for at diff ikke sku bli større enn nødvendig. Uten at jeg er heeelt på fast grunn, mener jeg dette fungerer fordi fila leses inn før funksjonene kalles, så StyledSafeLinkButton finnes når PreviewLightBox faktisk blir kjørt første gang (og at det da ikke hadde fungert hvis if-statementen inne i PreviewLightBox hadde vært utafor funksjon. Men jeg ser poenget; flytter!